apmpackage/apm/data_stream/app_logs/fields/ecs.yml

```diff
@@ -74,6 +74,8 @@
   name: ecs.version
 - external: ecs
   name: error.stack_trace
+- external: ecs
+  name: error.type
```

Comment:
Is the `error.type` really set for the `logs-apm.app_logs-*` datastream and not only for `logs-apm.error_logs-*`?